themes/src/main/resources/theme/base/login/messages/messages_en.properties

```diff
@@ -251,6 +251,7 @@ invalidPasswordExistingMessage=Invalid existing password.
 invalidPasswordBlacklistedMessage=Invalid password: password is blacklisted.
 invalidPasswordConfirmMessage=Password confirmation doesn''t match.
 invalidTotpMessage=Invalid authenticator code.
+notReusableTotpMessage=Cannot reuse this authenticator code. Please wait for the next one.
```

Comment:
Should most likely not leak that this was in fact a valid code.

Reply:
Ok, make sense. So, only show to the user 'Invalid authenticator code', right?